Documentation/hacking.md

```diff
@@ -10,24 +10,61 @@ For more information on the rkt internals, see the [`devel`](devel/) documentati
 rkt should be able to be built on any modern Linux system.
 For the most part the codebase is self-contained (e.g. all dependencies are vendored), but assembly of the stage1 requires some other tools to be installed on the system.
 
-### Build-time requirements
-
-* Linux 3.8+
-  * autoconf
-  * automake
-  * bc
-  * cpio
-  * gcc
-  * glibc development and static pieces (on Fedora/RHEL/Centos: glibc-devel and glibc-static packages, on Debian/Ubuntu libc6-dev package)
-  * gpg
-  * libacl development files (on Fedora/RHEL/Centos: libacl-devel, on Debian/Ubuntu libacl1-dev)
-  * make
-  * openssl
-  * patch
-  * realpath
-  * squashfs-tools
-  * wget
-* Go 1.4+
+### Build-time Requirements
+
+#### Basic
+
+* GNU Make
+* Go 1.4+ (ideally 1.5.3 or later)
+* autoconf
+* aclocal (usually a part of automake)
+* bash
+* git
+* glibc
+  * development headers
+  * the rkt binary links against the library
+* gofmt (usually distributed with Go)
+* govet (usually distributed with Go)
+* TrouSerS (only when TPM is enabled)
+  * development headers
+  * the rkt binary links against the library
+* gpg (when running functional tests)
+
+#### Additional requirements when also building any stage1 image flavor
+
+* glibc
+  * development headers
+  * the stage1 binaries link against the static library
+* libdl
+  * development headers
+  * the stage1 binaries link against the library
+* libacl
+  * development headers
+* C compiler
+
+#### Specific for the coreos/kvm flavor
+
+* cat
+* cpio
+* gzip
+* md5sum
+* mktemp
+* sort
+* unsquashfs
+* wget
+* gpg (optional, required when downloading the CoreOS PXE image during the build)
+
+#### Specific for the kvm flavor
+
+* patch
+* tar
+* xz
+* build requirements for kernel
+* build requirements for lkvm
+
+#### Specific for the src flavor
+
+* build requirements for systemd
```

Comment:
The documentation about the "Build verbosity" should not be part of the "src flavor" section